src/main/java/seedu/duke/TimeTable.java

```diff
@@ -3,59 +3,70 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.Calendar;
-import java.util.Locale;
+import java.util.NoSuchElementException;
 import java.util.Scanner;
+import java.util.Locale;
```

Comment:
The 'types of testing' section has just an ellipse.
Should you remove the section if it is not yet implemented?
A suggestion could be to state that the section would be implemented in the future.

![image](https://user-images.githubusercontent.com/60385689/97515344-d6ef6200-19cb-11eb-867f-089dbcb1fa7a.png)